src/VersionsCheck.php

```diff
@@ -50,11 +49,24 @@ public function checkPackages(ArrayRepository $distRepository, WritableRepositor
             if (\count($higherPackages) > 0) {
                 // Sort packages by highest version to lowest
                 usort($higherPackages, function (PackageInterface $p1, PackageInterface $p2) {
-                    return Comparator::compare($p1->getVersion(), '<', $p2->getVersion());
+                    if (Comparator::compare($p1->getVersion(), '=', $p2->getVersion())) {
+                        return 0;
+                    }
+                    if (Comparator::compare($p1->getVersion(), '<', $p2->getVersion())) {
+                        return 1;
+                    }
+                    return -1;
```

Comment:
Replace with ternary:
 `return Comparator::compare($p1->getVersion(), '<', $p2->getVersion()) ? 1 : -1;`
There shouldn't be any versions that are equal, right?